apps/passport-client/src/appHooks.ts

```diff
@@ -64,7 +66,13 @@ export function usePCDs(): PCD[] {
 
 export function usePCDsInFolder(folder: string): PCD[] {
   const pcds = usePCDCollection();
-  return [...pcds.getAllPCDsInFolder(folder)];
+  return [
+    ...pcds.getAllPCDsInFolder(folder).filter(
+      (pcd) =>
+        // Filter out PODTicketPCDs unless showPODTicketPCDs is true
+        pcd.type !== PODTicketPCDTypeName || appConfig.showPODTicketPCDs
```

Comment:
super minor thing but there's a slight chance the consumer of this function expects an unfiltered list, and could e.g. use the list returned by this function to do some sort of 'replace' operation, in which case the app may unintentionally delete pods. i'd say the risk is low but thought i'd point it out. i think ideally this filtering happens downstream of a hook like this, in a `useMemo` that gets mapped to a list of `JSX.Element` or something.